voicecommand/src/main/java/org/envirocar/voicecommand/service/ModelNetworkService.kt

```diff
@@ -0,0 +1,34 @@
+package org.envirocar.voicecommand.service
+
+import okhttp3.ResponseBody
+import org.envirocar.voicecommand.model.VoiceModel
+import retrofit2.Response
+import retrofit2.Retrofit
+import retrofit2.converter.gson.GsonConverterFactory
+import retrofit2.http.GET
+import retrofit2.http.Path
+import retrofit2.http.Streaming
+
+interface VoiceModelService {
+
+    @GET("models")
+    suspend fun getModelsList(): Response<List<VoiceModel>>
+
+    @Streaming
+    @GET("models/{name}")
+    suspend fun getModel(@Path("name") name: String): ResponseBody
+
+
+}
+
+object VoiceModelNetwork {
+
+
+    private val retrofit = Retrofit.Builder()
+        .baseUrl("http://ec2-15-206-186-192.ap-south-1.compute.amazonaws.com:8000/")
```

Comment:
You should make this URL configurable, since it may change in the future.

Reply:
Sure thing, I will just do that.
Should I store it as an environment variable (I would prefer this, as keeping the address public would have security concerns) or a string resource?
Or maybe just a BASE_URL variable as is the enviroCar API base address?
Let me know your thoughts.

Reply:
Sorry for the late response. I'd prefer to introduce an environment variable for it. I noticed, you have already commited the `BASE_URL` approach. If you are fine with having your own AWS service linked here, I will merge the request immediately. Otherwise, I would also wait if you want to implement the environment variable solution e.g. via `gradle.properties`.